anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/randomcutforest/RandomCutForestAnomalyDetector.java

```diff
@@ -110,6 +110,9 @@ public MappedMpoint classify(MappedMpoint mappedMpoint) {
             } else {
                 anomalyResult.setAnomalyLevel(AnomalyLevel.STRONG);
             }
+            anomalyResult.setMetric(mappedMpoint.getMpoint().getMetric());
+            anomalyResult.setObserved(new Double(mappedMpoint.getMpoint().getValue()));
```

Comment:
`new Double` shouldn't be required e.g. https://github.com/ExpediaDotCom/haystack-adaptive-alerting/blob/6f1ec5c8cabf2d78e5f2618f986e5289604d57f0/anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/EwmaAnomalyDetector.java#L148 But not a big deal.